src/hotel-search/Merger.js

```diff
@@ -116,6 +117,37 @@ HotelSearchClient.prototype = {
     }
   },
 
+  _mergeSortedRatesByBasePrice: function() {
+    var self = this,
+      cloneRates,
+      sortedCloneRates;
+
+    for (var hotelId in self.__hotelMap) {
+      if (self.__hotelMap[hotelId]) {
+        cloneRates = JSON.parse(JSON.stringify(self.__hotelMap[hotelId].rates));
```

Comment:
we already have a fews clone methods here, could you try to use that https://github.com/wego/wego-sdk-js/blob/master/src/utils.js#L2